### PR TITLE
Kill old running containers in hourly maintenance.

### PR DESCRIPTION
**Pull request by spxtr**
It's sharing a lot of code with `RemoveContainers`.

The problem is that we have a bunch of hanging containers on various Jenkins agents that still in the running state, so we never try to remove them. A week later, we try to remove their image, but it fails because it's in use.

This PR will make it kill any container that is still running after a day.

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/test-infra/1087)

### CODE REVIEW

**Comment by spxtr** [comment]
#1044

File: jenkins/hourly_maintenance.py [hunk 1]
```diff
@@ -35,6 +35,33 @@ def ContainerImages():
...
+    for line in subprocess.check_output([
+        'docker', 'ps', '-a',
+        '-f', 'status=running',
+        '--format={{.CreatedAt}}\t{{.ID}}',
+    ]).split('\n'):
+        if not line:
+            continue
+        created, name = line.split('\t')
+        fmt = 'YYYY-MM-dd HH:MM'
+        dt = datetime.datetime.strptime(created[:len(fmt)], '%Y-%m-%d %H:%M')
```
Comment (fejta) [inline jenkins/hourly_maintenance.py]: Might be worth encapsulating this inside a function since you're using it twice

**Comment by spxtr** [comment]
Ehh, I'm not sure how much refactoring is worth. Hopefully in the near-ish future we won't be running this anymore :)

**Pull request merged**